scripts/main/view.js

```diff
@@ -1489,6 +1508,51 @@ view.full_settings = {
 	},
 };
 
+view.notifications = {
+	init: function () {
+		multiselect.clearSelection();
+
+		view.notifications.title();
+		view.notifications.content.init();
+	},
+
+	title: function () {
+		lychee.setTitle("Notifications", false);
+	},
+
+	clearContent: function () {
+		lychee.content.html('<div class="settings_view"></div>');
+	},
+
+	content: {
+		init: function () {
+			view.notifications.clearContent();
+
+			$(".settings_view").append(
+				'<div class="setting_line"><p>' +
+					"Add your email below to enable receiving email notifications. " +
```

Comment:
So this will not be localized, correct?

Reply:
Not currently, but it can be if you'd like?

Reply:
That would be nice. :)